<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmDlg</class>
 <widget class="QDialog" name="AlarmDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>761</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>位地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>报警条件</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>消息</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="combo_language">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>语言:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>5</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_help">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>74</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radio_bit">
      <property name="text">
       <string>位监控</string>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_word">
      <property name="text">
       <string>字监控</string>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>报警名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bat">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>35</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>自动分配</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>全部重置</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>9</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_datatype">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>8</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>16位整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16位正整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位正整数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16位BCD码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位BCD码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32位浮点数</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
